### PR TITLE
Pre-publish changes

### PR DESCRIPTION
**Pull request by aleasims**
Some improvements before tagging&publishing this crate.

Some of them are required, like `license` field in `Cargo.toml`.

Also I'm trying to avoid warnings in build.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -2,6 +2,16 @@
 name = "gevulot-rs"
 version = "0.1.0"
```
Comment (tuommaki) [inline Cargo.toml:3]: Do we go with 0.1.0 or should we bump this e.g. to 0.1.3 at least to match **ember**?

**Review by tuommaki — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (aleasims) [reply Cargo.toml:3]: I don't know, should we?

Comment (aleasims) [reply Cargo.toml:3]: However this can be our public version of the network, so probably it makes sense

Comment (aleasims) [reply Cargo.toml]: Done

**Review by tuommaki — APPROVED** [review]

**Pull request merged**